.gitpod.yml

```diff
@@ -0,0 +1,28 @@
+image: gitpod/workspace-postgres
+# List the start up tasks. Learn more https://www.gitpod.io/docs/config-start-tasks/
+tasks:
+    - name: Dependencies # runs during prebuild
+      init: |
+          virtualenv venv && . ./venv/bin/activate
+          export PIP_USER=false
+          pip install poetry && poetry install
+          gp sync-done packages
+
+    - name: Database
+      init: |
+          gp sync-await packages
+          psql -c "CREATE USER open_event_user WITH PASSWORD 'opev_pass';"
+          psql -c "CREATE DATABASE oevent WITH OWNER open_event_user;"
+          cp .env.example .env
+          gp sync-done db
```

Comment:
What will be the admin credentials in this case?

Reply:
what should we do?
1. ask the user for entering and wait for them to enter
2. we should use a default admin

Reply:
Saving default credentials doesn't look like the correct approach as it will create problems with emails. @maze-runnar @daretobedifferent18  what do you think?

Reply:
What problem with emails??

Reply:
Email delivered to admins will get lost if we set a default email.